README.md

```diff
@@ -1,25 +1,29 @@
 # Deepgram Rust SDK
 
+[![Discord](https://dcbadge.vercel.app/api/server/xWRaCDBtW4?style=flat)](https://discord.gg/xWRaCDBtW4)
 [![CI](https://github.com/deepgram-devs/deepgram-rust-sdk/actions/workflows/ci.yaml/badge.svg?branch=main)](https://github.com/deepgram-devs/deepgram-rust-sdk/actions/workflows/ci.yaml)
```

Comment:
This URL should no longer be deepgram-devs

Reply:
Good catch. will fix.